src/test/java/showcase/neodymium/tests/responsive/MobileDeviceTest.java

```diff
@@ -0,0 +1,48 @@
+package showcase.neodymium.tests.responsive;
+
+import static com.codeborne.selenide.Condition.hidden;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+
+import org.junit.Test;
+
+import com.codeborne.selenide.Selenide;
+import com.xceptance.neodymium.module.statement.browser.multibrowser.Browser;
+import com.xceptance.neodymium.util.Neodymium;
+
+import showcase.neodymium.tests.AbstractTest;
+
+/**
+ * Most web sites provide their content in different resolutions for different devices.<br>
+ * In order to test a cell phone for example, the emulation of various mobile devices integrated in Chrome can be
+ * used.<br>
+ * <br>
+ * <b>REQUIRED CONFIGURATION</b> <i>config/browser.properties</i>:
```

Comment:
We use the 'REQUIRED CONFIGURATION section in the description mainly if a change by the user is required. For this test case we already set this up for the user. So we should just state that and where we configured/prepared this. So we give a hint where adjustments could be made.